tests/unit/.gitingore

```diff
@@ -1,2 +0,0 @@
-.DS_Store
```

Comment:
This is an unrelated change.  Remove it from this PR.

Reply:
@mccottry it is typo "gitingor" instead of "gitignore"
May be it make sense stay it here?

Reply:
Open another PR for this.  And create the appropriate task and subtask in Asana.